config/routes.rb

```diff
@@ -18,7 +18,9 @@
     end
     resources :access_requests, except: [:new, :create, :edit, :update]
 
-    resources :environments, only: [:index, :new, :create, :show, :destroy]
+    resources :environments, only: [
+      :index, :show, :new, :create, :edit, :update, :destroy
+    ]
```

Comment:
I'm mostly trying to keep lines under 80 chars, with some exceptions where readability is impacted.  This is because Misha likes lines under than length to allow 3 columns side-by-side in a text editor.  How do the rest of our team feel about this?